src/parse/asp/builtins.go

```diff
@@ -578,6 +579,10 @@ func packageName(s *scope, args []pyObject) pyObject {
 	return pyString(s.pkg.Name)
 }
 
+func subrepoName(s *scope, args []pyObject) pyObject {
+	return pyString("///" + s.pkg.SubrepoName)
```

Comment:
Note that `package_name` does not. Not sure if that's an inconsistency you want to care about, but it's worth noting.

Also, what if there is no subrepo name?

Ideally, I would use it like this: `subrepo_name() + package_name()` or `subrepo_name() + "//tools/k8s:helm"`.

If I want it to work in both subrepos and non-subrepos, I have to check if `subrepo_name() == "///"`.

Reply:
Haha my excuse is that this was a last minute PR at 7pm on a Monday.

Reply:
No worries! Thanks a lot!